resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+        </pre>
+                    </td>
+                </tr>
+                 <tr>
+                    <td><p>Request for an account that is closed with a closure reason as cancelled</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "3000000403"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_COMPLIANCE_CHECK_FAILED",
+          "message": "You cannot reinstate this account because the investor has failed a compliance check"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing an account ID that does not exist</p><p class="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "accountId": "0000000404"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+      operationId: reinstateLisaAccount
+      parameters:
+      - $ref: '#/components/parameters/acceptHeader'
+      - $ref: '#/components/parameters/contentTypeHeader'
+      - $ref: '#/components/parameters/authorizationHeader'
+      - $ref: '#/components/parameters/lisaManagerReferenceNumber'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/schemas/lifetimeIsaManagerAccountsReinstateAccountRequest'
+            examples:
+              example-1:
+                value:
+                  accountId: '8765432100'
+        required: true
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaManagerAccountsReinstateAccountResponse'
+              example:
+                status: 200
+                success: true
+                data:
+                  message: This account has been reinstated
+                  accountId: '8765432100'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/lifetimeIsaErrorResponse'
+              examples:
+                invalidPayload:
```

Comment:
This is missing a description. Please add, thanks.

                invalidPayload:
                  description: Bad Request
                  value:
                    code: BAD_REQUEST
                    message: Bad Request
                    errors:
                    - code: INVALID_FORMAT
                      message: Invalid format has been used
                      path: /accountId